samples/SmokeTest/src/main/java/com/azure/CosmosDB.java

```diff
@@ -0,0 +1,127 @@
+package com.azure;
+
+import com.azure.data.cosmos.CosmosClient;
+import com.azure.data.cosmos.CosmosContainer;
+import com.azure.data.cosmos.CosmosItemProperties;
+import com.azure.data.cosmos.FeedResponse;
+import com.azure.data.cosmos.FeedOptions;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
+
+public class CosmosDB {
+    private static CosmosClient client;
+    private static CosmosContainer container;
+    private static final String dbName = "JavaSolarSystem";
+    private static final String collectionName = "Planets";
+
+    private static void createDatabase() {
+        System.out.print("Creating database... ");
+        client.createDatabaseIfNotExists(dbName).block();
+        System.out.println("\tDONE.");
+    }
+
+    private static void createCollection() {
+        System.out.print("Creating Collection... ");
+        client.getDatabase(dbName).createContainer(collectionName, "/id").block();
+        container = client.getDatabase(dbName).getContainer(collectionName);
+        System.out.println("\tDONE.");
+    }
+
+    private static void createDocuments() {
+        System.out.println("Inserting Items... ");
+        Planet[] planets = new Planet[]{
+            new Planet(
+                "Earth",
+                false,
+                3959,
+                new Moon[]{
+                    new Moon("Moon")
+                }),
+            new Planet(
+                "Mars",
+                false,
+                2106,
+                new Moon[]{
+                    new Moon("Phobos"),
+                    new Moon("Deimos")
+                })
+        };
+
+        for (Planet planet : planets) {
```

Comment:
You can also use `Arrays.ofList()` for your array.

This can be more reactive with:
```java
private static Mono<Void> createDocuments() {
    return Mono.when(Flux.fromIterable(planets)).flatMap(contents -> {
          return container.createItem(planet);
    }));
}
```